resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosFineGrainedSchedulerBackend.scala

```diff
@@ -453,4 +453,8 @@ private[spark] class MesosFineGrainedSchedulerBackend(
       super.applicationId
     }
 
+  override def getNumSlots(): Int = {
+    // TODO support this method for MesosFineGrainedSchedulerBackend
```

Comment:
so this breaks barrier execution on mesos completely? (since available slot is 0 it will just fail)

Reply:
but finegrained is being deprecated...

Reply:
Only `MesosFineGrainedSchedulerBackend` shall break, we still support `MesosCoarseGrainedSchedulerBackend`

Reply:
@jiangxb1987 Could you create a JIRA and link here?